app/components/contact-page-components/ContactItem.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC } from "react";
+import Image, { StaticImageData } from "next/image";
+import { useTheme } from "next-themes";
+
+interface ContactItemProps {
+  contact: {
+    label: string;
+    imageSrc: StaticImageData;
+    imageSrcDark: StaticImageData;
+    info: string;
+  };
+}
+
+const ContactItem: FC<ContactItemProps> = ({ contact }) => {
```

Comment:
Try destructuring the props to see if you can remove the Functional Component import from React.